util/tc.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import absolute_import
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+import stat
+import urllib
```

Comment:
This needs to be `six.moves.urllib` for Python 2 and 3 compat.

Reply:
Pydoc says there is no `urlretrieve` in `six.moves.urllib` :/

Reply:
Done